error while running ./init.sh script in Quickstart Guide

While running going through this [guide ](https://github.com/Seagate/cortx-s3server/blob/main/docs/CORTX-S3%20Server%20Quick%20Start%20Guide.md)on an AWS CentOS machine.

and running this command
```
./init.sh -a
```

I run into this error below.


```
TASK [motr-build : build Lustre rpms] ***************************************************************************
task path: /root/cortx-s3server/third_party/motr/scripts/provisioning/roles/motr-build/tasks/main.yml:232
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~root && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp `"&& mkdir "` echo /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709 `" && echo ansible-tmp-1624575611.8-15896-25028441105709="` echo /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709 `" ) && sleep 0'
Using module file /usr/lib/python2.7/site-packages/ansible/modules/commands/command.py
<localhost> PUT /root/.ansible/tmp/ansible-local-141490RSm0z/tmp7A7u6t TO /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709/AnsiballZ_command.py
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709/ /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709/AnsiballZ_command.py && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709/AnsiballZ_command.py && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1624575611.8-15896-25028441105709/ > /dev/null 2>&1 && sleep 0'
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": [
        "rpmbuild",
        "-bb",
        "--without",
        "servers",
        "--without",
        "lustre-tests",
        "--define",
        "configure_args --disable-gss-keyring",
        "~/rpmbuild/SPECS/lustre.spec"
    ],
    "delta": "0:00:05.531877",
    "end": "2021-06-24 23:00:17.429606",
    "invocation": {
        "module_args": {
            "_raw_params": "rpmbuild -bb --without servers --without lustre-tests         --define 'configure_args --disable-gss-keyring'                  ~/rpmbuild/SPECS/lustre.spec\n",
            "_uses_shell": false,
            "argv": null,
            "chdir": null,
            "creates": "/root/rpmbuild/RPMS/x86_64/lustre-client-devel-*.rpm",
            "executable": null,
            "removes": null,
            "stdin": null,
            "stdin_add_newline": true,
            "strip_empty_ends": true,
            "warn": true
        }
    },
    "msg": "non-zero return code",
    "rc": 1,
    "start": "2021-06-24 23:00:11.897729",
    "stderr": "+ umask 022\n+ cd /root/rpmbuild/BUILD\n+ cd /root/rpmbuild/BUILD\n+ rm -rf lustre-2.12.5\n+ /usr/bin/gzip -dc /root/rpmbuild/SOURCES/lustre-2.12.5.tar.gz\n+ /usr/bin/tar -xf -\n+ STATUS=0\n+ '[' 0 -ne 0 ']'\n+ cd lustre-2.12.5\n+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .\n+ ln lustre/ChangeLog ChangeLog-lustre\n+ ln lnet/ChangeLog ChangeLog-lnet\n+ exit 0\n+ umask 022\n+ cd /root/rpmbuild/BUILD\n+ cd lustre-2.12.5\n+ cd /root/rpmbuild/BUILD/lustre-2.12.5\n+ export 'UTILS_CFLAGS=-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '\n+ UTILS_CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '\n+ export 'UTILS_LDFLAGS=-Wl,-z,relro '\n+ UTILS_LDFLAGS='-Wl,-z,relro '\n+ CONFIGURE_ARGS=--disable-gss-keyring\n+ '[' -n --disable-gss-keyring ']'\n++ echo --disable-gss-keyring\n++ sed -e 's/\"\\?--with-linux=[^ ][^ ]* \\?//'\n+ CONFIGURE_ARGS=--disable-gss-keyring\n++ echo --disable-gss-keyring\n++ sed -e 's/\"\\?--with-linux-obj=[^ ][^ ]* \\?//'\n+ CONFIGURE_ARGS=--disable-gss-keyring\n++ sed -e 's/\"\\?--with-kmp-moddir=[^ ][^ ]* \\?//'\n++ echo --disable-gss-keyring\n+ CONFIGURE_ARGS=--disable-gss-keyring\n+ CFLAGS='-g -O2 -Werror'\n+ export CFLAGS\n+ CXXFLAGS='-g -O2 -Werror'\n+ export CXXFLAGS\n+ FFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'\n+ export FFLAGS\n+ FCFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'\n+ export FCFLAGS\n+ LDFLAGS='-Wl,-z,relro '\n+ export LDFLAGS\n+ '[' 1 == 1 ']'\n+ '[' x86_64 == ppc64le ']'\n++ find . -name config.guess -o -name config.sub\n+ for i in '$(find . -name config.guess -o -name config.sub)'\n++ basename ./config/config.guess\n+ '[' -f /usr/lib/rpm/redhat/config.guess ']'\n+ /usr/bin/rm -f ./config/config.guess\n++ basename ./config/config.guess\n+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.guess ./config/config.guess\n+ for i in '$(find . -name config.guess -o -name config.sub)'\n++ basename ./config/config.sub\n+ '[' -f /usr/lib/rpm/redhat/config.sub ']'\n+ /usr/bin/rm -f ./config/config.sub\n++ basename ./config/config.sub\n+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.sub ./config/config.sub\n+ eval ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client\n++ ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client\nwhich: no mpicc in (/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)\ndirname: missing operand\nTry 'dirname --help' for more information.\nconfigure: error: Kernel source /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64 could not be found.\nerror: Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)\n    Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)",
    "stderr_lines": [
        "+ umask 022",
        "+ cd /root/rpmbuild/BUILD",
        "+ cd /root/rpmbuild/BUILD",
        "+ rm -rf lustre-2.12.5",
        "+ /usr/bin/gzip -dc /root/rpmbuild/SOURCES/lustre-2.12.5.tar.gz",
        "+ /usr/bin/tar -xf -",
        "+ STATUS=0",
        "+ '[' 0 -ne 0 ']'",
        "+ cd lustre-2.12.5",
        "+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .",
        "+ ln lustre/ChangeLog ChangeLog-lustre",
        "+ ln lnet/ChangeLog ChangeLog-lnet",
        "+ exit 0",
        "+ umask 022",
        "+ cd /root/rpmbuild/BUILD",
        "+ cd lustre-2.12.5",
        "+ cd /root/rpmbuild/BUILD/lustre-2.12.5",
        "+ export 'UTILS_CFLAGS=-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '",
        "+ UTILS_CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '",
        "+ export 'UTILS_LDFLAGS=-Wl,-z,relro '",
        "+ UTILS_LDFLAGS='-Wl,-z,relro '",
        "+ CONFIGURE_ARGS=--disable-gss-keyring",
        "+ '[' -n --disable-gss-keyring ']'",
        "++ echo --disable-gss-keyring",
        "++ sed -e 's/\"\\?--with-linux=[^ ][^ ]* \\?//'",
        "+ CONFIGURE_ARGS=--disable-gss-keyring",
        "++ echo --disable-gss-keyring",
        "++ sed -e 's/\"\\?--with-linux-obj=[^ ][^ ]* \\?//'",
        "+ CONFIGURE_ARGS=--disable-gss-keyring",
        "++ sed -e 's/\"\\?--with-kmp-moddir=[^ ][^ ]* \\?//'",
        "++ echo --disable-gss-keyring",
        "+ CONFIGURE_ARGS=--disable-gss-keyring",
        "+ CFLAGS='-g -O2 -Werror'",
        "+ export CFLAGS",
        "+ CXXFLAGS='-g -O2 -Werror'",
        "+ export CXXFLAGS",
        "+ FFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'",
        "+ export FFLAGS",
        "+ FCFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'",
        "+ export FCFLAGS",
        "+ LDFLAGS='-Wl,-z,relro '",
        "+ export LDFLAGS",
        "+ '[' 1 == 1 ']'",
        "+ '[' x86_64 == ppc64le ']'",
        "++ find . -name config.guess -o -name config.sub",
        "+ for i in '$(find . -name config.guess -o -name config.sub)'",
        "++ basename ./config/config.guess",
        "+ '[' -f /usr/lib/rpm/redhat/config.guess ']'",
        "+ /usr/bin/rm -f ./config/config.guess",
        "++ basename ./config/config.guess",
        "+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.guess ./config/config.guess",
        "+ for i in '$(find . -name config.guess -o -name config.sub)'",
        "++ basename ./config/config.sub",
        "+ '[' -f /usr/lib/rpm/redhat/config.sub ']'",
        "+ /usr/bin/rm -f ./config/config.sub",
        "++ basename ./config/config.sub",
        "+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.sub ./config/config.sub",
        "+ eval ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client",
        "++ ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client",
        "which: no mpicc in (/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)",
        "dirname: missing operand",
        "Try 'dirname --help' for more information.",
        "configure: error: Kernel source /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64 could not be found.",
        "error: Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)",
        "    Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)"
    ],
    "stdout": "Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.LbJuXm\nExecuting(%build): /bin/sh -e /var/tmp/rpm-tmp.JsKD4p\n‘/usr/lib/rpm/redhat/config.guess’ -> ‘./config/config.guess’\n‘/usr/lib/rpm/redhat/config.sub’ -> ‘./config/config.sub’\nchecking build system type... x86_64-unknown-linux-gnu\nchecking host system type... x86_64-unknown-linux-gnu\nchecking target system type... x86_64-unknown-linux-gnu\nchecking for a BSD-compatible install... /bin/install -c\nchecking whether build environment is sane... yes\nchecking for a thread-safe mkdir -p... /bin/mkdir -p\nchecking for gawk... gawk\nchecking whether make sets $(MAKE)... yes\nchecking whether make supports nested variables... yes\nchecking whether UID '0' is supported by ustar format... yes\nchecking whether GID '0' is supported by ustar format... yes\nchecking how to create a ustar tar archive... gnutar\nchecking whether to enable maintainer-specific portions of Makefiles... yes\nchecking for gcc... gcc\nchecking whether the C compiler works... yes\nchecking for C compiler default output file name... a.out\nchecking for suffix of executables... \nchecking whether we are cross compiling... no\nchecking for suffix of object files... o\nchecking whether we are using the GNU C compiler... yes\nchecking whether gcc accepts -g... yes\nchecking for gcc option to accept ISO C89... none needed\nchecking for style of include used by make... GNU\nchecking dependency style of gcc... none\nchecking how to print strings... printf\nchecking for a sed that does not truncate output... /bin/sed\nchecking for grep that handles long lines and -e... /bin/grep\nchecking for egrep... /bin/grep -E\nchecking for fgrep... /bin/grep -F\nchecking for ld used by gcc... /bin/ld\nchecking if the linker (/bin/ld) is GNU ld... yes\nchecking for BSD- or MS-compatible name lister (nm)... /bin/nm -B\nchecking the name lister (/bin/nm -B) interface... BSD nm\nchecking whether ln -s works... yes\nchecking the maximum length of command line arguments... 1572864\nchecking whether the shell understands some XSI constructs... yes\nchecking whether the shell understands \"+=\"... yes\nchecking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop\nchecking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop\nchecking for /bin/ld option to reload object files... -r\nchecking for objdump... objdump\nchecking how to recognize dependent libraries... pass_all\nchecking for dlltool... no\nchecking how to associate runtime and link libraries... printf %s\\n\nchecking for ar... ar\nchecking for archiver @FILE support... @\nchecking for strip... strip\nchecking for ranlib... ranlib\nchecking command to parse /bin/nm -B output from gcc object... ok\nchecking for sysroot... no\nchecking for mt... no\nchecking if : is a manifest tool... no\nchecking how to run the C preprocessor... gcc -E\nchecking for ANSI C header files... yes\nchecking for sys/types.h... yes\nchecking for sys/stat.h... yes\nchecking for stdlib.h... yes\nchecking for string.h... yes\nchecking for memory.h... yes\nchecking for strings.h... yes\nchecking for inttypes.h... yes\nchecking for stdint.h... yes\nchecking for unistd.h... yes\nchecking for dlfcn.h... yes\nchecking for objdir... .libs\nchecking if gcc supports -fno-rtti -fno-exceptions... no\nchecking for gcc option to produce PIC... -fPIC -DPIC\nchecking if gcc PIC flag -fPIC -DPIC works... yes\nchecking if gcc static flag -static works... no\nchecking if gcc supports -c -o file.o... yes\nchecking if gcc supports -c -o file.o... (cached) yes\nchecking whether the gcc linker (/bin/ld -m elf_x86_64) supports shared libraries... yes\nchecking whether -lc should be explicitly linked in... no\nchecking dynamic linker characteristics... GNU/Linux ld.so\nchecking how to hardcode library paths into programs... immediate\nchecking whether stripping libraries is possible... yes\nchecking if libtool supports shared libraries... yes\nchecking whether to build shared libraries... yes\nchecking whether to build static libraries... yes\nchecking dependency style of gcc... none\nchecking for rpmbuild... /bin/rpmbuild\nconfigure: Lustre base checks\n==============================================================================\nchecking whether to configure just enough for make dist... no\nchecking if this distro uses dpkg... no\nchecking for ranlib... (cached) ranlib\nchecking for ld... /bin/ld -m elf_x86_64\nchecking for objdump... (cached) objdump\nchecking for strip... (cached) strip\nchecking size of unsigned long long... 8\nchecking for -Wno-format-truncation support... yes\nchecking for -Wno-stringop-truncation support... yes\nchecking for -Wno-stringop-overflow support... yes\nchecking whether to use OSD addon... no\nchecking whether to build Lustre docs... no\nchecking whether to build Lustre manpages... yes\nchecking whether to build Lustre utilities... yes\nchecking whether to build Lustre tests... no\nchecking whether to build Lustre client support... yes\nchecking whether mpitests can be built... no\nchecking whether to build Linux kernel modules... yes (linux-gnu)\nchecking for Linux sources... /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64\nchecking for /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64... no\n\n\nRPM build errors:",
    "stdout_lines": [
        "Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.LbJuXm",
        "Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.JsKD4p",
        "‘/usr/lib/rpm/redhat/config.guess’ -> ‘./config/config.guess’",
        "‘/usr/lib/rpm/redhat/config.sub’ -> ‘./config/config.sub’",
        "checking build system type... x86_64-unknown-linux-gnu",
        "checking host system type... x86_64-unknown-linux-gnu",
        "checking target system type... x86_64-unknown-linux-gnu",
        "checking for a BSD-compatible install... /bin/install -c",
        "checking whether build environment is sane... yes",
        "checking for a thread-safe mkdir -p... /bin/mkdir -p",
        "checking for gawk... gawk",
        "checking whether make sets $(MAKE)... yes",
        "checking whether make supports nested variables... yes",
        "checking whether UID '0' is supported by ustar format... yes",
        "checking whether GID '0' is supported by ustar format... yes",
        "checking how to create a ustar tar archive... gnutar",
        "checking whether to enable maintainer-specific portions of Makefiles... yes",
        "checking for gcc... gcc",
        "checking whether the C compiler works... yes",
        "checking for C compiler default output file name... a.out",
        "checking for suffix of executables... ",
        "checking whether we are cross compiling... no",
        "checking for suffix of object files... o",
        "checking whether we are using the GNU C compiler... yes",
        "checking whether gcc accepts -g... yes",
        "checking for gcc option to accept ISO C89... none needed",
        "checking for style of include used by make... GNU",
        "checking dependency style of gcc... none",
        "checking how to print strings... printf",
        "checking for a sed that does not truncate output... /bin/sed",
        "checking for grep that handles long lines and -e... /bin/grep",
        "checking for egrep... /bin/grep -E",
        "checking for fgrep... /bin/grep -F",
        "checking for ld used by gcc... /bin/ld",
        "checking if the linker (/bin/ld) is GNU ld... yes",
        "checking for BSD- or MS-compatible name lister (nm)... /bin/nm -B",
        "checking the name lister (/bin/nm -B) interface... BSD nm",
        "checking whether ln -s works... yes",
        "checking the maximum length of command line arguments... 1572864",
        "checking whether the shell understands some XSI constructs... yes",
        "checking whether the shell understands \"+=\"... yes",
        "checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop",
        "checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop",
        "checking for /bin/ld option to reload object files... -r",
        "checking for objdump... objdump",
        "checking how to recognize dependent libraries... pass_all",
        "checking for dlltool... no",
        "checking how to associate runtime and link libraries... printf %s\\n",
        "checking for ar... ar",
        "checking for archiver @FILE support... @",
        "checking for strip... strip",
        "checking for ranlib... ranlib",
        "checking command to parse /bin/nm -B output from gcc object... ok",
        "checking for sysroot... no",
        "checking for mt... no",
        "checking if : is a manifest tool... no",
        "checking how to run the C preprocessor... gcc -E",
        "checking for ANSI C header files... yes",
        "checking for sys/types.h... yes",
        "checking for sys/stat.h... yes",
        "checking for stdlib.h... yes",
        "checking for string.h... yes",
        "checking for memory.h... yes",
        "checking for strings.h... yes",
        "checking for inttypes.h... yes",
        "checking for stdint.h... yes",
        "checking for unistd.h... yes",
        "checking for dlfcn.h... yes",
        "checking for objdir... .libs",
        "checking if gcc supports -fno-rtti -fno-exceptions... no",
        "checking for gcc option to produce PIC... -fPIC -DPIC",
        "checking if gcc PIC flag -fPIC -DPIC works... yes",
        "checking if gcc static flag -static works... no",
        "checking if gcc supports -c -o file.o... yes",
        "checking if gcc supports -c -o file.o... (cached) yes",
        "checking whether the gcc linker (/bin/ld -m elf_x86_64) supports shared libraries... yes",
        "checking whether -lc should be explicitly linked in... no",
        "checking dynamic linker characteristics... GNU/Linux ld.so",
        "checking how to hardcode library paths into programs... immediate",
        "checking whether stripping libraries is possible... yes",
        "checking if libtool supports shared libraries... yes",
        "checking whether to build shared libraries... yes",
        "checking whether to build static libraries... yes",
        "checking dependency style of gcc... none",
        "checking for rpmbuild... /bin/rpmbuild",
        "configure: Lustre base checks",
        "==============================================================================",
        "checking whether to configure just enough for make dist... no",
        "checking if this distro uses dpkg... no",
        "checking for ranlib... (cached) ranlib",
        "checking for ld... /bin/ld -m elf_x86_64",
        "checking for objdump... (cached) objdump",
        "checking for strip... (cached) strip",
        "checking size of unsigned long long... 8",
        "checking for -Wno-format-truncation support... yes",
        "checking for -Wno-stringop-truncation support... yes",
        "checking for -Wno-stringop-overflow support... yes",
        "checking whether to use OSD addon... no",
        "checking whether to build Lustre docs... no",
        "checking whether to build Lustre manpages... yes",
        "checking whether to build Lustre utilities... yes",
        "checking whether to build Lustre tests... no",
        "checking whether to build Lustre client support... yes",
        "checking whether mpitests can be built... no",
        "checking whether to build Linux kernel modules... yes (linux-gnu)",
        "checking for Linux sources... /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64",
        "checking for /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64... no",
        "",
        "",
        "RPM build errors:"
    ]
}
msg:  non-zero return code
stderr:  + umask 022
+ cd /root/rpmbuild/BUILD
+ cd /root/rpmbuild/BUILD
+ rm -rf lustre-2.12.5
+ /usr/bin/gzip -dc /root/rpmbuild/SOURCES/lustre-2.12.5.tar.gz
+ /usr/bin/tar -xf -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd lustre-2.12.5
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ ln lustre/ChangeLog ChangeLog-lustre
+ ln lnet/ChangeLog ChangeLog-lnet
+ exit 0
+ umask 022
+ cd /root/rpmbuild/BUILD
+ cd lustre-2.12.5
+ cd /root/rpmbuild/BUILD/lustre-2.12.5
+ export 'UTILS_CFLAGS=-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '
+ UTILS_CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  '
+ export 'UTILS_LDFLAGS=-Wl,-z,relro '
+ UTILS_LDFLAGS='-Wl,-z,relro '
+ CONFIGURE_ARGS=--disable-gss-keyring
+ '[' -n --disable-gss-keyring ']'
++ echo --disable-gss-keyring
++ sed -e 's/"\?--with-linux=[^ ][^ ]* \?//'
+ CONFIGURE_ARGS=--disable-gss-keyring
++ echo --disable-gss-keyring
++ sed -e 's/"\?--with-linux-obj=[^ ][^ ]* \?//'
+ CONFIGURE_ARGS=--disable-gss-keyring
++ sed -e 's/"\?--with-kmp-moddir=[^ ][^ ]* \?//'
++ echo --disable-gss-keyring
+ CONFIGURE_ARGS=--disable-gss-keyring
+ CFLAGS='-g -O2 -Werror'
+ export CFLAGS
+ CXXFLAGS='-g -O2 -Werror'
+ export CXXFLAGS
+ FFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'
+ export FFLAGS
+ FCFLAGS='-g -O2 -Werror -I/usr/lib64/gfortran/modules'
+ export FCFLAGS
+ LDFLAGS='-Wl,-z,relro '
+ export LDFLAGS
+ '[' 1 == 1 ']'
+ '[' x86_64 == ppc64le ']'
++ find . -name config.guess -o -name config.sub
+ for i in '$(find . -name config.guess -o -name config.sub)'
++ basename ./config/config.guess
+ '[' -f /usr/lib/rpm/redhat/config.guess ']'
+ /usr/bin/rm -f ./config/config.guess
++ basename ./config/config.guess
+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.guess ./config/config.guess
+ for i in '$(find . -name config.guess -o -name config.sub)'
++ basename ./config/config.sub
+ '[' -f /usr/lib/rpm/redhat/config.sub ']'
+ /usr/bin/rm -f ./config/config.sub
++ basename ./config/config.sub
+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.sub ./config/config.sub
+ eval ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client
++ ./configure --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --disable-gss-keyring --disable-tests --enable-utils --enable-modules --disable-ldiskfs --disable-server --without-zfs --disable-snmp --disable-gss --disable-gss-keyring --with-systemdsystemunitdir=/usr/lib/systemd/system --with-linux=/lib/modules/3.10.0-1127.19.1.el7.x86_64/source --with-linux-obj=/lib/modules/3.10.0-1127.19.1.el7.x86_64/build --with-kmp-moddir=extra/lustre-client
which: no mpicc in (/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)
dirname: missing operand
Try 'dirname --help' for more information.
configure: error: Kernel source /usr/src/kernels/3.10.0-1127.19.1.el7.x86_64 could not be found.
error: Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)
    Bad exit status from /var/tmp/rpm-tmp.JsKD4p (%build)
[WARNING]: Failure using method (v2_runner_on_failed) in callback plugin
(<ansible.plugins.callback.pretty_print.CallbackModule object at 0x7f78d4609ad0>): 'ascii' codec can't encode
character u'\u2018' in position 107: ordinal not in range(128)
Callback Exception:
  File "/usr/lib/python2.7/site-packages/ansible/executor/task_queue_manager.py", line 381, in send_callback
    method(*new_args, **kwargs)
   File "/root/cortx-s3server/third_party/motr/scripts/provisioning/plugins/pretty_print.py", line 117, in v2_runner_on_failed
    self.pretty_print(result._result)
   File "/root/cortx-s3server/third_party/motr/scripts/provisioning/plugins/pretty_print.py", line 53, in pretty_print
    output = self._format(data[rec]).replace("\\n", "\n")
   File "/root/cortx-s3server/third_party/motr/scripts/provisioning/plugins/pretty_print.py", line 79, in _format
    return str(output)

PLAY RECAP ******************************************************************************************************
localhost                  : ok=55   changed=2    unreachable=0    failed=1    skipped=7    rescued=0    ignored=2
```

--- a/docs/CORTX-S3 Server Quick Start Guide.md
+++ b/docs/CORTX-S3 Server Quick Start Guide.md
@@ -371,6 +371,8 @@ Refer to our [CORTX Contribution Guide](https://github.com/Seagate/cortx/blob/ma
 Please refer to the [Support](https://github.com/Seagate/cortx/blob/main/SUPPORT.md) section to reach out to us with your questions, contributions, and feedback.
 
 Tested by:
+    
+- Aug 19, 2021: Patrick Hession (patrick.hession@seagate.com) in CentOS 7.8.2003 on VMWare Workstation Pro 16 and bare metal CentOS server
 - Aug 3, 2021: Daniar Kurniawan (daniar@uchicago.edu) in CentOS 7.8.2003 on a Chameleon node (type=compute_skylake).
 - Aug 3, 2021: Meng Wang (wangm12@uchicago.edu) in CentOS 7.8.2003 on a Chameleon node (type=compute_skylake).
 - Aug 3, 2021: Akhil Bhansali (akhil.bhansali@seagate.com) on SSC cloud VM (Centos 7.9)

